File: CP/code/intro/uva_11364.cc
#include <cstdio>

using namespace std;

int main(int argc, char *argv[])
{
    int t, n;

    scanf("%d", &t);
    while (t-- > 0) {
        scanf("%d", &n);
        int max = 0, min = 100;
        int tmp;
        while (n-- > 0) {
            scanf("%d", &tmp);
            if (tmp > max) max = tmp;
            if (tmp < min) min = tmp;
        }
        printf("%d\n", 2 * (max - min));
    }
    return 0;
}

File: CP/code/intro/uva_11364.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        int t, n;

        Scanner in = new Scanner(System.in);
        t = in.nextInt();
        while (t-- > 0) {
            n = in.nextInt();
            int max = 0, min = 100;
            int tmp;
            while (n-- > 0) {
                tmp = in.nextInt();
                if (tmp > max) max = tmp;
                if (tmp < min) min = tmp;
            }
            System.out.println(2 * (max - min));
        }
    }
    
}
